posts/2021-04-30-dev-mode-with-aot.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Open Liberty development mode startup improved with AOT"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/dsouzai
+author_github: https://github.com/dsouzai
+seo-title: Open Liberty development mode startup improved with AOT - OpenLiberty.io
+seo-description: Improved Open liberty deveopment mode startup with AOT compilation using Eclipse OpenJ9.
+blog_description: "Improved Open liberty deveopment mode startup with AOT compilation using Eclipse OpenJ9."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Open Liberty development mode startup improved with AOT
```

Comment:
one last comment before I make edits- For the title-it seems like the user benefit is faster dev mode startup- do you want to emphasize AOT in that context or the use of Open J9? Based on the conclusion of the post, it seems like the takeaway is that OpenJ9 now has faster startup with OL. If that's the case, I'd recommend changing the title to something like

Faster startup in Open Liberty dev mode with OpenJ9 0.26.0